frontend/src/v5/store/containers/containers.helpers.ts

```diff
@@ -27,11 +27,7 @@ import { getNullableDate } from '@/v5/helpers/getNullableDate';
 import filesize from 'filesize';
 import { formatMessage } from '@/v5/services/intl';
 
-export const filterContainers = (federations: IContainer[], filterQuery: string) => (
-	federations.filter((
-		{ name, code, type },
-	) => [name, code, type].join('').toLowerCase().includes(filterQuery.trim().toLowerCase()))
-);
+export const CONTAINERS_SEARCH_FIELDS = ['code', 'type', 'name', 'desc', 'latestRevision'];
```

Comment:
We should probably do something similar for projects, because you're currently able to search by projectId which can be confusing as the id isn't shown. So you end up getting projects showing up when you'd expect them to be filtered out